client/src/api/AuthService.ts

```diff
@@ -0,0 +1,34 @@
+import axios, { AxiosResponse } from 'axios'
+import { IAxiosDataResponse } from './types'
+
+export default class AuthService {
+    static async validateUserToken(
+        token: string
+    ): Promise<AxiosResponse<IAxiosDataResponse>> {
+        const response = await axios.get<IAxiosDataResponse>(
+            'http://localhost:5000/api/auth/me',
+            {
+                headers: {
```

Comment:
you should have a general handler function called `requestApi` which handles all the requests under the hood. for example, adds token

Reply:
Fixed. Used a axios interceptors to solve this problem